src/renderer/components/+landing-page/workspace-cluster.store.ts

```diff
@@ -0,0 +1,55 @@
+import { WorkspaceId } from "../../../common/workspace-store";
+import { Cluster } from "../../../main/cluster";
+import { clusterStore } from "../../../common/cluster-store";
+import { ItemObject, ItemStore } from "../../item.store";
+import { autobind } from "../../utils";
+
+export class ClusterItem implements ItemObject {
+  constructor(public cluster: Cluster) {}
+  
+  getName() {
+    return this.cluster.name;
+  }
+
+  getId() {
+    return this.cluster.id;
+  }
```

Comment:
Having these also as getters would allow for some other code to be good.
```suggestion
  get name() {
    return this.cluster.name;
  }

  getName() {
    return this.name;
  }
  
  get id() {
    return this.cluster.id;
  }
  
  get clusterId() {
    return this.cluster.id;
  }

  getId() {
    return this.id;
  }
```